functions.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * Connects to the database, setting the attribute and executing the query by fetching all the data
+ *
+ * @param $collectionName refers to the database name, referenced through the function connect_db on the index.php file
+ *
+ * @return the new PDO, setting up the connection between the php file and the database
+ *
+ */
+
+function connect_db_return_collection($collectionName)
```

Comment:
split this function into two functions. one function for connecting to the db, and a different one for getting the data from the database